api/index.py

```diff
@@ -165,17 +174,22 @@ def get(self):
                 )
             )
 
-        # TODO: change x to be the width of the fabric
-        container_max_x = 2000
-        container_max_y = 2000
+        container_max_x = resolution_manager._get_bounding_box_width_limit()
```

Comment:
Im pretty sure for the _ notation with functions you're only supposed to use them within the class itself